mistune/renderers.py

```diff
@@ -198,3 +198,66 @@ def list(self, text, ordered, level, start=None):
 
     def list_item(self, text, level):
         return '<li>' + text + '</li>\n'
+
+
+class TextRenderer(BaseRenderer):
+    NAME = 'text'
+    IS_TREE = False
+
+    def __init__(self):
+        super(TextRenderer, self).__init__()
+
+    def text(self, text):
+        return text
+
+    def heading(self, text, level):
+        return '#' * level + ' ' + text.strip() + '\n\n'
+
+    def list(self, text, ordered, level, start=None):
+        assert not ordered
+        return text
+
+    def list_item(self, text, level):
+        return ' ' * (level-1) + '- ' + text + '\n'
+
+    def block_text(self, text):
+        return text
+
+    def thematic_break(self):
+        return '\n---\n\n'
+
+    def paragraph(self, text):
+        return text.strip('\n') + '\n\n'
+
+    def link(self, link, text=None, title=None):
+        return f'[{text}]({link})'
+
+    def block_quote(self, text):
+        return '\n'.join(['> ' + line for line in text.strip().splitlines()]) + '\n'
+
+    def newline(self):
+        return '\n'
+
+    def block_code(self, code, info=None):
+        return f"```{info or ''}\n" + code.strip('\n') + "\n```\n\n"
+
+    def codespan(self, text):
+        return f'`{text}`'
+
+    def strong(self, text):
+        return '**' + text + '**'
```

Comment:
Since signac intends to use this in a terminal context, it might be a nice touch to use terminal bolding. I've used this in other applications with several terminals / multiple platforms, so I think it works reliably.
```suggestion
        return '\033[1m' + text + '\033[0m'
```

Reply:
And maybe `TerminalRenderer` is a better name than `TextRenderer`? Not sure.

Reply:
I recognize that I didn't fully commit to either text or terminal rendering both in the code and the branch name. 😄 

I'm happy to fix that before moving forward.